pkgs/top-level/python-packages.nix

```diff
@@ -4196,6 +4196,28 @@ in modules // {
       description = "pytest plugin with mechanisms for caching across test runs";
     };
   };
+  
+  pytestdjango = buildPythonPackage rec {
+    name = "pytest-django-${version}";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pytest-django/${name}.tar.gz";
+      sha256 = "1mmc7zsz3dlhs6sx4sppkj1vgshabi362r1a8b8wpj1qfximpqcb";
+    };
+    
+    # doing this to allow depending packages to find
+    # pytest's binaries
+    pytest = self.pytest;
```

Comment:
Actually the above example is far more complex than it has to be... all you have to consider is I might want to do e.g.

```
someCommand = writeScript "some-command" ''
  ...
  ${xyzCode.pytestdjango.pytest}
  ...
'';
```

In some very package-specific script and know that it's going to refer to the exact binary it's built against, not swayed by whatever the user happens to have in their `$PATH` at runtime.